### PR TITLE
Swap out UK org types for US-specific org types

### PR DESCRIPTION
**Pull request by stvnrlly**
This adds in "federal" and "state" as organization types in the DB, to match up with https://github.com/GSA/notifications-admin/pull/59. It should also enforce those choices on API calls, but I haven't checked that directly.

Because the SQL commands don't (seem to) cascade cleanly, we should wait to merge this until we transition to a fresh cloud.gov space. If we want to merge before then, some additional DB cleanup is needed.

### CODE REVIEW

File: migrations/versions/0376_add_org_names.py
Comment (rahearn) [inline migrations/versions/0376_add_org_names.py:20]: Should we have a line to remove unused org types from the db?

**Review by rahearn — COMMENTED** [review]
Is there a reason to leave the UK specific org types in the db?

File: migrations/versions/0376_add_org_names.py
Comment (stvnrlly) [reply migrations/versions/0376_add_org_names.py:20]: Yeah, removing them didn't seem to cascade cleanly, so I left them in to avoid breaking the demo. But we shouldn't have that issue when we set up a fresh instance, so I'll add it and we can hold off on merging until after we get our new space.

**Comment by stvnrlly** [comment]
Okay, obviously I rebased incorrectly, but otherwise this is ready for review

**Review by rahearn — APPROVED** [review]
🎉 I'd love to delete that one comment before merging, but no need for a re-review after it's done.

**Pull request merged**
